.github/workflows/test.yml

```diff
@@ -35,9 +35,14 @@ jobs:
             {"kernel": "LATEST", "runs_on": ["ubuntu-latest", "self-hosted"], "arch": "x86_64", "toolchain": "${{ needs.llvm-toolchain.outputs.llvm }}"},
             {"kernel": "LATEST", "runs_on": ["z15", "self-hosted"], "arch": "s390x", "toolchain": "gcc"},
           ]
+          self_hosted_repos = [
+            "kernel-patches/bpf",
+            "kernel-patches/vmtest",
+          ]
 
-          if "${{ github.repository_owner }}" != "kernel-patches":
-            # Outside of kernel-patches, remove the self-hosted label and skip
+          # Only a few repository within "kernel-patches" use self-hosted runners.
```

Comment:
I had this set up this way originally, mostly because I though just the trailing part of the repository could be accessed, and then stripped it as redundant. Happy to add it back, I don't have strong opinions.

Side note: Who is hiding behind the bot account?

Reply:
> Side note: Who is hiding behind the bot account?

In the past it was mykola-lysenko when he forgot to swap accounts. I suppose it could be anybody with access to the bot account, though (and I don't know who else that includes).